src/js/modules/infragistics.ui.notifier.js

```diff
@@ -559,14 +559,19 @@
 			return template;
 		},
 		_setNewContent: function (nc) {
-			var newContent = nc;
+			var newContent = nc, iconContainer = this.css.iconContainer, icon = "";
 			if (nc instanceof $) {
 				newContent = nc.html();
 			} else if (typeof nc === "object") {
 				newContent = nc.innerHTML;
 			}
-			newContent = newContent.replace(/\{0\}/g, this.css.iconContainer)
-				.replace(/\{1\}/g, this.options.showIcon ? this.css[ this.options.state + "Icon" ] : "")
+			if (this.options.showIcon) {
+				icon = this.css[ this.options.state + "Icon" ];
```

Comment:
why do we have a capitalized `css` class name?

Reply:
That'd be the option name - `this.css.errorIcon`, `this.css.successIcon` which is in line with naming in other controls :)